scripts/upgrade/upgrade-utils.ts

```diff
@@ -0,0 +1,8 @@
+import fs from "fs";
+import path from "path";
+
+export function readUpgradeParameters() {
+  const filePath = path.join(__dirname, "upgrade-parameters-mainnet.json");
```

Comment:
We'll first need to upgrade the hoodi testnet. Instead of hardcoding, we can add an optional parameter that will override any hardcode and use whatever is specified in it.